templates/repo/diff/whitespace_dropdown.tmpl

```diff
@@ -2,30 +2,30 @@
 	{{svg "gitea-whitespace"}}
 	{{svg "octicon-triangle-down" 14 "dropdown icon"}}
 	<div class="menu">
-		<a class="item" href="?style={{if .IsSplitStyle}}split{{else}}unified{{end}}&whitespace=show-all">
+		<a class="item" href="?style={{if .IsSplitStyle}}split{{else}}unified{{end}}&show-outdated={{$.ShowOutdatedComments}}&whitespace=show-all">
 			<label class="gt-pointer-events-none">
 				<input class="gt-mr-3 gt-pointer-events-none" type="radio"{{if eq .WhitespaceBehavior "show-all"}} checked{{end}}>
 				{{.locale.Tr "repo.diff.whitespace_show_everything"}}
 			</label>
 		</a>
-		<a class="item" href="?style={{if .IsSplitStyle}}split{{else}}unified{{end}}&whitespace=ignore-all">
+		<a class="item" href="?style={{if .IsSplitStyle}}split{{else}}unified{{end}}&show-outdated={{$.ShowOutdatedComments}}&whitespace=ignore-all">
 			<label class="gt-pointer-events-none">
 				<input class="gt-mr-3 gt-pointer-events-none" type="radio"{{if eq .WhitespaceBehavior "ignore-all"}} checked{{end}}>
 				{{.locale.Tr "repo.diff.whitespace_ignore_all_whitespace"}}
 			</label>
 		</a>
-		<a class="item" href="?style={{if .IsSplitStyle}}split{{else}}unified{{end}}&whitespace=ignore-change">
+		<a class="item" href="?style={{if .IsSplitStyle}}split{{else}}unified{{end}}&show-outdated={{$.ShowOutdatedComments}}&whitespace=ignore-change">
 			<label class="gt-pointer-events-none">
 				<input class="gt-mr-3 gt-pointer-events-none" type="radio"{{if eq .WhitespaceBehavior "ignore-change"}} checked{{end}}>
 				{{.locale.Tr "repo.diff.whitespace_ignore_amount_changes"}}
 			</label>
 		</a>
-		<a class="item" href="?style={{if .IsSplitStyle}}split{{else}}unified{{end}}&whitespace=ignore-eol">
+		<a class="item" href="?style={{if .IsSplitStyle}}split{{else}}unified{{end}}&show-outdated={{$.ShowOutdatedComments}}&whitespace=ignore-eol">
```

Comment:
Please make all these links a stable order, e.g. style,whitespace,show-outdated.

Reply:
Sort order should now be stable across all links.